opentelemetry-resource-detectors/README.md

```diff
@@ -0,0 +1,18 @@
+![OpenTelemetry — An observability framework for cloud-native software.][splash]
+
+[splash]: https://raw.githubusercontent.com/open-telemetry/opentelemetry-rust/main/assets/logo-text.png
+
+# OpenTelemetry Contrib Resource Detectors
```

Comment:
nit - remove contrib ?

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/49/commits/98052034707af7285864ea267c899b9b8b632e08
Thank you